contracts/Insurance.sol

```diff
@@ -26,8 +24,8 @@ contract Insurance is IInsurance {
     // The insurance pool funding rate calculation can be refactored to have 0.00000570775
     // as the constant in front; see getPoolFundingRate for the formula
     // (182.648 / 8) * (5 ** 2) * (1 / (10_000 ** 2)) = 0.00000570775
-    // 0.00000570775 as a WAD = 570775 * (10 ** 7)
-    uint256 private constant INSURANCE_FUNDING_RATE_FACTOR = 570775 * (10**7);
+    // 0.00000570775 as a WAD = 5.70775e12
+    uint256 private constant INSURANCE_FUNDING_RATE_FACTOR = 5.70775e12;
```

Comment:
Not sure if I necessarily find this easier to read though :thinking:

Reply:
Yeah not a clear improvement, however most of our other constants use scientific notation so I suppose there is the benefit of consistency.